Batch #1

notdoctorq opened the pull request:
Each symbol has a single commit now.

issus commented:
Comparators:
Mostly just file name style.

**SCH - COMPARATOR - TI LM2903DR.SCHLIB** - Package is actually D not DR - the R denotes its on a Reel. The symbol should suit D, DGK, JG, P, PS, or PW Package, so I would leave out the package for this symbol. Pinout looks good.

**SCH - COMPARATOR - TI TL331IDBVR.SCHLIB** - either use VCC/GND or V+/V- as the other symbols do. You have V+ and GND in this symbol, the datasheet specifies V-/GND - so I would go V- as it could be a negative voltage. Filename should be SCH - COMPARATOR - TI TL331IDBV.SCHLIB as above.

**SCH - COMPARATOR - TI LM393.SCHLIB** - I would suggest a rename to SCH - COMPARATOR - TI LMx93.SCHLIB as the symbol can be utilised for LM193, LM293, LM293A, LM393, LM393A, LM2903, LM2903V, LM393B, LM2903B.

**SCH - COMPARATOR - TI LM311DR.SCHLIB** - I would leave out the package designation as it is suitable for D, JG, P, PS, or PW Packages. TI Datasheet has this as 'LMx11' as it covers 111/211/311 parts. Therefore, I suggest a rename to SCH - COMPARATOR - TI LMx11.SCHLIB.

**SCH - COMPARATOR - TI LM339APWR.SCHLIB** - Suggest rename to 'SCH - COMPARATOR - TI LMx39.SCHLIB' inline with comments above. I'd also suggest adding a duplicate for LM2901, which uses the same symbol pinout :)

**SCH - COMPARATOR - ST LM393ADT.SCHLIB** - Suggest a rename to 'SCH - COMPARATOR - ST LMx93.SCHLIB' - pinout covers multiple parts/packages.

**SCH - COMPARATOR - ON SEMI LM339DR2G.SCHLIB** - Package suffix not required, all packages have the same pinout. Input 1/2 labels are swapped. Part A has IN1 labels for IN2 pins, and Part B has IN2 labels for IN1 pins.

**SCH - COMPARATOR - TI TPS3700DDCR.SCHLIB** - Package is DDC, not DDCR. All other comparators have used V+/V-, normally I go with the datasheet pin labels rather than any convention for these... so do we change this symbol to V+/- or the others to VCC/VDD and GND/VSS where the datasheet has different naming?

**SCH - COMPARATOR - MICROCHIP MCP6541T-I-LT.SCHLIB** - Suggest a rename to 'SCH - COMPARATOR - MICROCHIP MCP6541 5PIN.SCHLIB' and then prioritise adding symbols for U and R variants if you have time, to save confusion due to the multiple _completely different_ pin outs this part number offers!

When renaming files, don't forget to rename the parts within.

issus commented:
Opamps:

**SCH - OPAMP - TI LMV824MTX-NOPB.SCHLIB** - Rename to SCH - OPAMP - TI LMV82x 14PIN.SCHLIB. Pinout looks good.

**SCH - OPAMP - TI LM321MFX-NOPB.SCHLIB** - Rename SCH - OPAMP - TI LM321.SCHLIB - only available in DBV. Pinout is good.

**SCH - OPAMP - TI LMV601MG-NOPB.SCHLIB** - Filename doesnt need anything after 601, there's only 1 package option. SHDN doesnt need to be rotated.

notdoctorq commented:
(I have implemented all changes, and changed color scheme to the voted on yellow)
> Comparators:
> Mostly just file name style.
> 
> **SCH - COMPARATOR - TI LM2903DR.SCHLIB** - Package is actually D not DR - the R denotes its on a Reel. The symbol should suit D, DGK, JG, P, PS, or PW Package, so I would leave out the package for this symbol. Pinout looks good.
> 
> **SCH - COMPARATOR - TI TL331IDBVR.SCHLIB** - either use VCC/GND or V+/V- as the other symbols do. You have V+ and GND in this symbol, the datasheet specifies V-/GND - so I would go V- as it could be a negative voltage. Filename should be SCH - COMPARATOR - TI TL331IDBV.SCHLIB as above.
> 
> **SCH - COMPARATOR - TI LM393.SCHLIB** - I would suggest a rename to SCH - COMPARATOR - TI LMx93.SCHLIB as the symbol can be utilised for LM193, LM293, LM293A, LM393, LM393A, LM2903, LM2903V, LM393B, LM2903B.
> 
> **SCH - COMPARATOR - TI LM311DR.SCHLIB** - I would leave out the package designation as it is suitable for D, JG, P, PS, or PW Packages. TI Datasheet has this as 'LMx11' as it covers 111/211/311 parts. Therefore, I suggest a rename to SCH - COMPARATOR - TI LMx11.SCHLIB.
> 
> **SCH - COMPARATOR - TI LM339APWR.SCHLIB** - Suggest rename to 'SCH - COMPARATOR - TI LMx39.SCHLIB' inline with comments above. I'd also suggest adding a duplicate for LM2901, which uses the same symbol pinout :)
> Done
> **SCH - COMPARATOR - ST LM393ADT.SCHLIB** - Suggest a rename to 'SCH - COMPARATOR - ST LMx93.SCHLIB' - pinout covers multiple parts/packages.
> Done
> **SCH - COMPARATOR - ON SEMI LM339DR2G.SCHLIB** - Package suffix not required, all packages have the same pinout. Input 1/2 labels are swapped. Part A has IN1 labels for IN2 pins, and Part B has IN2 labels for IN1 pins.
> 
> **SCH - COMPARATOR - TI TPS3700DDCR.SCHLIB** - Package is DDC, not DDCR. All other comparators have used V+/V-, normally I go with the datasheet pin labels rather than any convention for these... so do we change this symbol to V+/- or the others to VCC/VDD and GND/VSS where the datasheet has different naming?
> Per our conversation, this one gets GND. 
> **SCH - COMPARATOR - MICROCHIP MCP6541T-I-LT.SCHLIB** - Suggest a rename to 'SCH - COMPARATOR - MICROCHIP MCP6541 5PIN.SCHLIB' and then prioritise adding symbols for U and R variants if you have time, to save confusion due to the multiple _completely different_ pin outs this part number offers!
> 
> When renaming files, don't forget to rename the parts within.
